app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -9,15 +9,15 @@
 
 @import 'partials/_shame';
 
-@import 'colours/_colours-palette';
-@import 'colours/_colours-organisations';
-@import 'colours/_colour';
-@import 'colours/_colours-applied';
-
+@import 'lib/govuk-frontend/govuk/all';
 @import 'partials/_step-by-step-navigation';
 @import 'partials/_lists.scss';
 @import 'partials/_layout.scss';
 
 .text-right {
   text-align: right;
-}
\ No newline at end of file
+}
+
+$govuk-assets-path: '/customs-movements/assets/';
```

Comment:
I am not sure that variable definition should goes before import

Reply:
I don't think it matters, we could try this approach